packages/site/src/components/Navigation/TableOfContents.tsx

```diff
@@ -112,8 +112,8 @@ export const TableOfContents = ({
 }) => {
   const [open] = useNavOpen();
   const config = useSiteManifest();
-  const { folder } = useParams();
-  const resolvedProjectSlug = projectSlug || folder;
+  const { folder, project } = useParams();
+  const resolvedProjectSlug = projectSlug || (folder ?? project);
```

Comment:
This is still pretty broken as  it depends on what you name your files in the theme. But slight improvement towards project terminology.